### PR TITLE
docs: add line-heights

### PR DESCRIPTION
**Pull request by wiese**
Bug: [T256855](https://phabricator.wikimedia.org/T256855)

### CODE REVIEW

**Comment by wiese** [comment]
Want to group this with the rest of the font stuff but does not make much sense while there are multiple PRs in progress competing for those incrementing ids.

**Comment by wiese** [comment]
rebased

File: docs/src/10-font-line-height.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<Meta title="Design Tokens|Font line-height" />
+
+## line-height
+
+<TokenTable
+    tokens={font[ 'line-height' ]}
+    valueCell={( value ) => (
+        <div>
+            <pre>{ value }</pre>
+            <div className='lineHeight' style={{ lineHeight: value }}>demo</div>
```
Comment (tzhelyazkova) [inline docs/src/10-font-line-height.stories.mdx:15]:
```suggestion
            <div className='lineHeight' style={{ lineHeight: value }}>
            The quick brown fox jumps over the lazy dog
            </div>
```
I think the visuals can benefit from a longer sentence. It makes the line height visible between lines as well.
Like here https://www.lightningdesignsystem.com/design-tokens/#category-line-height

Comment (wiese) [reply docs/src/10-font-line-height.stories.mdx]: Hmm. I guess for me it became less clear, or maybe I misunderstood.

Comment (tzhelyazkova) [reply docs/src/10-font-line-height.stories.mdx]: Ok, we can ask UX first, before we merge this one.
I think line-height is most important in paragraphs.

**Comment by wiese** [comment]
I also removed the horizontal alignment which did not make sense with an intentional multiline.

**Pull request merged**